bin/inasafe

```diff
@@ -0,0 +1,318 @@
+#!/usr/bin/python
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool developed by AusAid -
+**CLI implementation for inasafe project.**
+
+Contact : jannes@kartoza.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+    Note :
+        cli: command line interface
+"""
+
+__author__ = 'Jannes Engelbrecht'
+__date__ = '16/04/15'
+from docopt import docopt, DocoptExit
```

Comment:
Please pay attention to import order:

* builtins
* third party
* project

See https://github.com/AIFDR/inasafe/wiki/Coding-Standards#ordering-of-imports